var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -239,50 +383,58 @@ def install(self, spec, prefix):
             else:
                 cmake_prefix_path = join_path(spec['dyninst'].prefix)
                 with working_dir('build', create=True):
-
-                    # python_vers=join_path(spec['python'].version[:2])
-                    # '-DOPENMPI_DIR=%s'            % openmpi_prefix_path,
-                    # '-DMVAPICH_DIR=%s'            % mvapich_prefix_path,
-                    # '-DMPICH_DIR=%s'              % spec['mpich'].prefix,
-                    # '-DMPICH2_DIR=%s'             % spec['mpich2'].prefix,
-                    # '-DBoost_NO_SYSTEM_PATHS=TRUE',
-                    # '-DBOOST_ROOT=%s'             % spec['boost'].prefix,
-                    # '-DOPENMPI_DIR=%s'            % spec['openmpi'].prefix,
-
-                    python_vers = '%d.%d' % spec['python'].version[:2]
-
+                    python_vers = format(spec['python'].version.up_to(2))
+                    python_pv = '/python' + python_vers
+                    python_pvs = '/libpython' + python_vers + '.so'
```

Comment:
@davydden @alalazo How do you specify library suffixes again? I can't remember if this changed recently. This will work fine on Linux, but Mac usually uses a different suffix.